container/rkt/helpers.go

```diff
@@ -20,17 +20,58 @@ import (
 	"path"
 	"strings"
 
+	rktapi "github.com/coreos/rkt/api/v1alpha"
 	"github.com/golang/glog"
+	"golang.org/x/net/context"
 )
 
 type parsedName struct {
 	Pod       string
 	Container string
 }
 
-func verifyName(name string) (bool, error) {
-	_, err := parseName(name)
-	return err == nil, err
+func verifyPod(name string) (bool, error) {
+	pod, err := cgroupToPod(name)
+
+	if err != nil || pod == nil {
+		return false, err
+	}
+
+	// anything handler can handle is also accepted
+	// accept cgroups that are sub the pod cgroup, except "system.slice"
+	accept := !strings.HasSuffix(name, "/system.slice")
```

Comment:
Maybe more documentation here explaining why we don't accpet `/system.slice`. Or link any issues related to this?